runtime/src/lib.rs

```diff
@@ -134,6 +101,11 @@ pub type UncheckedExtrinsic =
 /// Extrinsic type that has already been checked.
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, TypeInfo, Copy, MaxEncodedLen)]
+pub enum Region {
+	Europe,
+}
+
```

Comment:
Did you try to declare the type in the same way as `SS58Prefix` below?

Reply:
Yes. But that did not change the generated Typescript type.